### ISSUE TITLE
Polarization leakage in Epoch of Reionization windows: I. LOFAR observations of the 3C196 field

### ISSUE THREAD

**Issue by Trienko** [ratt-ru/reading-list#11]
Important paper I think. Maybe for one of the next sessions.

http://arxiv.org/abs/1503.01644

**Comment by Chuneeta** [comment]
Yes Trienko, this is the paper I am planning to talk in the next session.

**Comment by Trienko** [comment]
Hi Ridhima,

Cool...

Trienko

On Tue, Mar 10, 2015 at 9:50 AM, Nunhokee Chuneeta <notifications@github.com

> wrote:
> 
> Yes Trienko, this is the paper I am planning to talk in the next session.
>
